src/color-scale/color-scale.js

```diff
@@ -144,7 +147,11 @@ const Self = class ColorScale extends NudeElement {
 				}
 
 				return colors;
-			}
+			},
+			dependencies: ["colors", "steps", "space", "coords"],
+		},
+		coords: {
+			type: String,
```

Comment:
Why is this `type: String`? At the very least it should be an array or object.

Reply:
Oh I see, because it just passes itself verbatim to `<color-swatch>`. You don't need `type: String` then, just don't include a type at all. In the future we'll probably need to parse it in `<color-scale>` though, since it could be useful for `<color-scale>` to have summary stats on these coords. But for now just passing it through is ok.